### DISCUSSION TITLE
The Legend of Zelda: Link's Awakening 2019 - Stabilise FPS (potato version)

### DISCUSSION THREAD

**Discussion by ytayeb943** [StevensND/switch-port-mods#29] [Mod Request]
There are a few FPS stabilisers for the Link's Awakening remake, but they were only designed for emulated copies, they always crash when running on native hardware. Would it be possible to release a potato version FPS stabiliser for Link's Awakening, like was done for [Echoes of Wisdom](https://gamebanana.com/mods/545054)?

**Comment by StevensND** [comment]
> There are a few FPS stabilisers for the Link's Awakening remake, but they were only designed for emulated copies, they always crash when running on native hardware. Would it be possible to release a potato version FPS stabiliser for Link's Awakening, like was done for [Echoes of Wisdom](https://gamebanana.com/mods/545054)?

I haven't tried the mods for that game on Switch yet.

Also: 

1. Even though the engine is the same, the code is different so I don't know if it will work.

2. I didn't do Stabilize FPS Potato Version. I did  Normal Version using another method in comparison with Fl4sh's mod.

**Reply by ytayeb943** [reply]
Hmm okay, I will contact them and ask their thoughts on modding Link's Awakening

**Reply by StevensND** [reply]
> Hmm okay, I will contact them and ask their thoughts on modding Link's Awakening

Fl4sh didn't do Stablize FPS for Link's Awakening if that's what you're thinking.

The person who did it it's not longer doing mods.

**Comment by StevensND** [comment]
@ytayeb943 I tested the Stabilize FPS mod uploaded on Internet using my own Switch and yes ... game is crashing. 

However I did this mod and I want to know if you have FPS drops or not. I played and tested the mod on Handheld mode until the first boss fight and 0 FPS drops. Remember that you must have your Switch with overclock for better FPS stability.

[LA - 60FPS (IPS).zip](https://github.com/user-attachments/files/17610138/LA.-.60FPS.IPS.zip)

**Reply by ytayeb943** [reply]
I tested your patch in handheld mode and it is a lot smoother now. My overclock settings are 460 GPU / 1600 MEM and I only get an occasional frame drop; in my testing it's mainly when I use the running boots or when I visit the swamp area. But in general, the game runs much smoother with this patch. Thanks!

**Reply by StevensND** [reply]
I can't give you a good advice for OC due to I just use [NxVenom](https://github.com/CatcherITGF/NX-Venom) and one friend helped me to setup everything.

Anyway [here's a guide](https://rentry.co/howtoget60fps) about OC too.

I'm glad it works better now for you :)
